plugins/arDominionB5Plugin/modules/digitalobject/templates/_metadata.php

```diff
@@ -20,90 +20,94 @@
             </button>
           </h3>
           <div id="preservation-collapse" class="accordion-collapse collapse show" aria-labelledby="preservation-heading">
-            <div class="accordion-body">
+            <div class="accordion-body p-0">
               <?php if ($showOriginalFileMetadata) { ?>
 
-                <div class="digital-object-metadata-header">
-                  <h3><?php echo __('Original file'); ?> <i class="fa fa-archive<?php if (!$canAccessOriginalFile) { ?> inactive<?php } ?>" aria-hidden="true"></i></h3>
-                </div>
+                <div class="<?php echo render_b5_show_field_css_classes(); ?>">
+
+                  <h3 class="<?php echo render_b5_show_label_css_classes(); ?>"><?php echo __('Original file'); ?> <i class="fa fa-archive text-dark<?php if (!$canAccessOriginalFile) { ?> inactive text-muted<?php } ?>" aria-hidden="true"></i></h3>
```

Comment:
I have not checked how it looks, but you may want to remove the space between the heading text and the icon and use `ms-1` (or 2) class in the icon.

Reply:
And other icons below.